taqueria-sdk/index.ts

```diff
@@ -100,6 +101,9 @@ export const getArch = (): LikeAPromise<string, TaqError> => {
 	}
 };
 
+export const getFlextesaImage = (arch: 'linux/arm64/v8' | 'linux/amd64'): string =>
```

Comment:
I'm not sure whether this should be in the SDK or not. Seems like a SoC violation, no?

Reply:
Should I put it in the plugin instead?

Reply:
Though I think it's general enough to be put here 🤔

Reply:
I could go either way on this, so I'll leave it up to you. I just wanted to probe your thoughts.